2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 10 Line 1_
```markdown
## Permutation feature importance
```
I really like your idea here, but here is how I would approach it (take from it what you find useful):
* I think it is too big a challenge for the audience (and especially time-consuming) to code all three other options. I would have you present both shuffle and mean, then let them do min and max.
* I know I asked you made a plot of how the variables are changed after permutation (and I stand by that), but maybe just do it for a single variable as an example. Now there are too many plots and becomes overloaded. If you do shuffle and mean as I propose, you could showcase them in the same plot, which I find a nice comparison (plus I think for shuffle in particular it is nice to see the "noise" effect that you mention in the theory).


_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
I do not fully agree. For your first point, if it takes too much time people can just copy the answers. Also the min and max are essentially a direct copy of the mean, so should be easy. For your second point, again people are coding themselves. So if they do not want any plots, they can remove them.